content/en/docs/guides/compression.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Compression
+description: >-
+    How to use compression to reduce the bandwidth used by gRPC
```

Comment:
Better to say something like "how to compress the data sent over the wire while using gRPC"